text/0019-precland-indication.md

```diff
@@ -0,0 +1,41 @@
+  * Start date: 2022-11-07
+  * Contributors: Jacob Dahl <dahl.jakejacob@gmail.com>
+  * Related issues: https://github.com/mavlink/mavlink/pull/1858
+
+# Summary
+
+When interacting with a [Landing Target](https://mavlink.io/en/services/landing_target.html) it is useful to the user to know the current state of the
+precision maneuver -- ie Precision Land.
+
+
+# Motivation
+
+A failed precision landing could result in a vehicle crash, for example if the vehicle is landing on a small landing pad and loses the target during descent. In this scenario, it is critical that the operator knows the current status of the precision landing maneuver so that they can take control of the drone in the case the maneuver fails. The failure case for precision land (in PX4) is to fallback to a
+normal land, and this may not be desirable under certain circumstances. It is also useful to know if the precision landing system can see a target,
+regardless of whether the vehicle is actively performing a precision maneuver. This allows an operator to at-a-glance confirm if the precision targetting
+system is working properly.
+
+# Detailed Design
+
+MAVLink already supports the concept of "Sensor Status" via the MAV_SYS_STATUS_SENSOR and MAV_SYS_STATUS_SENSOR_EXTENDED bitmasks in the [SYS_STATUS](https://mavlink.io/en/messages/common.html#SYS_STATUS) message. I have implemented this functionality using MAV_SYS_STATUS_SENSOR_XY_POSITION_CONTROL but I propose that we create a new purpose built bitmask MAV_SYS_STATUS_SENSOR_PRECISION_TARGETTING.
```

Comment:
I see what you're doing with MAV_SYS_STATUS_SENSOR_XY_POSITION_CONTROL below, but what's the proposal for  MAV_SYS_STATUS_SENSOR_PRECISION_TARGETTING (I might be missing something obvious)

Reply:
The proposal is to just add this new flag MAV_SYS_STATUS_SENSOR_PRECISION_TARGETTING since "XY position control" isn't really accurate -- to me "xy position control" makes me think of an optical flow or VIO system. There are already flags for vision and optical flow tho... just seems like a lot of these flags are unused and not well thought out in the first place. A flag that clearly says "precision targetting" in the name seems more appropriate, and would probably be acceptable (in terms of language) to both px4/ardu peeps